src/leveled_runner.erl

```diff
@@ -602,16 +556,6 @@ foldobjects(SnapFun, Tag, KeyRanges, FoldObjFun, DeferredFetch,
                                                                 LastModRange,
                                                                 LimitByCount)
                 end,
-            AfterFun = 
-                fun() ->
-                    ok = leveled_penciller:pcl_close(LedgerSnapshot),
-                    case DeferredFetch of 
-                        {true, false} ->
```

Comment:
Here we actively did not close the JournalSnapshot.. but is that still the case in the refactored code?